packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -145,15 +145,30 @@ export interface FunctionProps {
 
   /**
    * What security group to associate with the Lambda's network interfaces.
+   * This property is being deprecated, consider using securityGroups instead.
    *
    * Only used if 'vpc' is supplied.
+   * Only use this or the securityGroups prop, not both at once
```

Comment:
This suggests the user can use securityGroup. 

Instead change this to `Use securityGroups property instead. Function constructor will throw an error if both are specified`

Reply:
Done